services/gatekeeper/3.8.2/release/release.yaml

```
@@ -26,10 +26,24 @@ spec:
   releaseName: kommander-gatekeeper
   valuesFrom:
     - kind: ConfigMap
-      name: gatekeeper-3.8.1-d2iq-defaults
+      name: gatekeeper-3.8.2-d2iq-defaults
     - kind: ConfigMap
       name: gatekeeper-overrides
       optional: true
+  postRenderers:
+    - kustomize:
+        # Gatekeeper set the min TLS version to 1.3 in 3.8.0 but FIPS enabled clusters do not support TLS 1.3,
+        # In 3.8.1, the --tls-min-version flag is not configurable so we
```

I found the tiniest nit here imaginable. BUT, I do think this is a comma situation here.

```suggestion
        # In 3.8.1, the --tls-min-version flag is not configurable, so we
```

---

my IDE even told me to do that but I ignored it lol, fixed in 62361137b9a23dcca432a792ab7b12d5aa682ed9